scipy/stats/_stats_py.py

```diff
@@ -2671,12 +2675,25 @@ def zscore(a, axis=0, ddof=0, nan_policy='propagate'):
         The z-scores, standardized by mean and standard deviation of
         input array `a`.
 
+    See Also
+    --------
+    numpy.mean : Arithmetic average
+    numpy.std : Arithmetic standard deviation
+    gzscore : Geometric standard score
```

Comment:
This is making the CI fail. I am not sure why though. Could you check this locally?

Reply:
FWIW: I can reproduce the warning locally, running either Sphinx 4.5.0 or 5.3.0.  As an experiment, I tried adding `gzscore` to the `See Also` section of a bunch of other functions, and I got the warning from `gmean`, `hmean` and `zscore`, but not from `pmean`, `tmean`, `mode` and others.  Go figure.  Do we have any resident Sphinx gurus who can take a look at this?

Reply:
Might be an order thing if the function is defined before or after? Does it fix it if we instead use a full resolution with `scipy.stats.gzscore`?

Reply:
Doc is OK when building `scipy.stats.zscore`
https://output.circle-artifacts.com/output/job/0b43ce1a-05db-4959-a5d5-9101dd342737/artifacts/0/html/reference/generated/scipy.stats.zscore.html

Doc is KO when building `scipy.stats.mstats.zscore`
https://output.circle-artifacts.com/output/job/0b43ce1a-05db-4959-a5d5-9101dd342737/artifacts/0/html/reference/generated/scipy.stats.mstats.zscore.html

Reply:
Let's use `scipy.stats.zscore` as we in the end want to slowly move away from `mstats`.